pkg/plugins/scms/github/releaseType.go

```diff
@@ -2,13 +2,13 @@ package github
 
 // ReleaseType specifies accepted GitHub Release type
 type ReleaseType struct {
-	// Draft enable/disable GitHub draft release
+	// "Draft" enable/disable GitHub draft release
```

Comment:
Are these comments visible by end suer through JSON schema, or is it development only?

Reply:
Yes it's visible to endusers via www.updatecli.io and jsonschema. 
Comments must be one line before the variable